ada_feeding_perception/package.xml

```diff
@@ -11,6 +11,7 @@
   <test_depend>ament_flake8</test_depend>
   <test_depend>ament_pep257</test_depend>
   <test_depend>python3-pytest</test_depend>
+  <test_depend>opencv-python</test_depend>
```

Comment:
Fascinating that this had to be a `test_depend` and nothing else. 

No need to change this, I just find it interesting